webapp/src/modules/wallet/utils.js

```diff
@@ -23,18 +23,83 @@ export function getWalletSagaOptions() {
       ),
       new Marketplace(env.get('REACT_APP_MARKETPLACE_CONTRACT_ADDRESS')),
       new EstateRegistry(env.get('REACT_APP_ESTATE_REGISTRY_CONTRACT_ADDRESS')),
-      ...getLandAuctionContract(),
+      ...getLandAuctionContracts(),
       ...getMortgageContracts()
     ],
     eth
   }
 }
 
-function getLandAuctionContract() {
+function getLandAuctionContracts() {
   const { LANDAuction } = contracts
 
+  // ZIL
+  const ZILToken = new contracts.ERC20Token(
```

Comment:
I think this is perfectly ok, but what do you think about having a`getNewERC20Token(name, address)` in this file and using it for this tokens as well as for the `RCNToken` ?

Reply:
done ✅